cocos/spine/skeleton.ts

```diff
@@ -364,6 +364,16 @@ export class Skeleton extends UIRenderer {
         }
     }
 
+    /**
+     * @en get defaultSkin.
+     * @zh 获取 defaultSkin 的值.
+     * @method getDefaultSkin
+     * @return {string}
+     */
+    getDefaultSkin (): string {
+        return this.defaultSkin;
```

Comment:
It's better to define it as getter.
First, you can declare serialize variable  '_defaultSkin', it can be declared like this:
``
    @serializable
    @formerlySerializedAs('defaultSkin')
    protected _defaultSkin = '';
``

Second, you can declare getter method:
``
  private set defaultSkin (name: string) {
        this._defaultSkin = name;
    }

    get defaultSkin (): string {
        return this._defaultSkin;
    }
``

Reply:
对，内部可以直接使用_defaultSkin, 外部使用getter的方式

Reply:
@finscn 可以从这个pr 遴选相关修改
https://github.com/cocos/cocos-engine/pull/16635